app/assets/javascripts/discourse/app/templates/navigation/categories.hbs

```diff
@@ -1,6 +1,6 @@
 <DSection @bodyClass="navigation-categories" @class="navigation-container">
   <DNavigation
-    @filterType="categories"
+    @filterMode="categories"
```

Comment:
For consistency with other `<DNavigation` invocations. Doesn't ultimately affect the result, since in this case `filterMode` is the same as `filterType`